{"ray": {"sid": "5c0aa4a523064557b632084e56a473e7", "uid": null, "qid": "e7b6f67bc42949a1844d6d4bc869ce62", "rid": null, "bars": {"default": {"percent": "100", "remaining": "0"}}, "messages": [{"type": "ERROR", "content": "process - failed executing: [e7b6f67bc42949a1844d6d4bc869ce62]\nTraceback (most recent call last):\n  File \"C:\\Users\\avina\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python311\\site-packages\\openfabric_pysdk\\app\\worker.py\", line 90, in process\n    output = execution_callback_function(data, ray, self.state)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\avina\\Downloads\\openfabric-ai-software-engineer 2\\openfabric-ai-software-engineer\\main.py\", line 152, in execute\n    agent_executor = get_model_state()\n                     ^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\avina\\Downloads\\openfabric-ai-software-engineer 2\\openfabric-ai-software-engineer\\main.py\", line 141, in get_model_state\n    verbose=False, output_parser=NewAgentOutputParser())\n                                 ^^^^^^^^^^^^^^^^^^^^\nNameError: name 'NewAgentOutputParser' is not defined\n", "created_at": "2023-12-30T18:10:23.839751"}], "status": "FAILED", "created_at": "2023-12-30T18:10:22.742535", "updated_at": "2023-12-30T18:10:23.839751", "finished": true}, "in": {"text": ["string"]}, "out": {}}